### ISSUE TITLE
TypeError

### ISSUE THREAD

**Issue by the4ndy** [rllynch/smartthings_cli#5]
```
PS> smartthings_cli query switch ALL
Traceback (most recent call last):
  File "C:\Python27\Scripts\smartthings_cli-script.py", line 11, in <module>
    load_entry_point('smartthings-cli==1.0', 'console_scripts', 'smartthings_cli')()
  File "build\bdist.win32\egg\smartthings_cli\smartthings_cli.py", line 305, in main
  File "build\bdist.win32\egg\smartthings_cli\smartthings_cli.py", line 128, in get_status
TypeError: string indices must be integers
```

Also is there support for Lights?

**Comment by txwindsurfer** [comment]
I am getting the same TypeError.  Please let me know if there is a fix or a simple error I am making.

**Comment by rllynch** [comment]
Basic on/off light functionality is supported by treating them as switches.

I can't reproduce the TypeError but I'm suspecting it's the same issue as #4 which I just fixed in a955f48f814dedfd0d77d3a42efe7c920087776a.  Try pulling the latest code and see if that helps.

**Comment by txwindsurfer** [comment]
Thanks Richard!  I have been using your excellent code and great work to
get establish a connection that allows a Lutron Homeworks QS system to
issues a smartthings command.  I got the telnet session working with a
direct connection to the Homeworks processor (pretty sure Lutron Connect
Hub won't work as it appears to communicate based only on MAC addresses).
I was not able to get the twisted/soap connection to smartthings, but I
used an IFTTT webhook to send a smartthings command from the python program
when there is a button push on a Lutron keypad.  This was all that I wanted
to accomplish, but now I am interested to see if I can get your code to
work fully with the Lutron Homeworks QS telnet interface since Lutron does
not currently support a Connect Hub connection in smartthings.  I will
continue to work as I have time and if you have any interest, I would
appreciate your time and help but I know the smartthings_cli code is
depreciated due to the official Lutron support of the Caseta Hub for
smartthings.
